docs/products/flink/howto/rest-api-cheatsheet.rst

```diff
@@ -0,0 +1,82 @@
+Flink REST API examples cheatsheet
```

Comment:
Why are we encouraging use of the API here? Is it expected to be a common use case for Flink users? I'd also like either a bit more of an explanation of the API or a link to our API docs and overview information to get started, get a token, etc.

I would consider converting these into curl commands that could be copied and pasted.

Reply:
This was built when no CLI or Console was available. Happy to remove since we have it documented in our API